code/modules/antagonists/changeling/powers/tiny_prick.dm

```diff
@@ -11,8 +11,6 @@
 	click_override = new(CALLBACK(src, PROC_REF(try_to_sting)))
 
 /datum/action/changeling/sting/Destroy(force, ...)
-	if(cling.owner.current.middleClickOverride == click_override)
-		cling.owner.current.middleClickOverride = null
```

Comment:
I put this here so if the changeling is de-antaged and they happen to have a prepared sting, it will be cleaned up. What's the runtime this is causing?

Reply:
No current so everything goes up in flames, but yeah I can account for that

Reply:
done